### PR TITLE
Set next_attempt in the next day for WA tiers limit hit

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: backends/rapidpro/status.go [hunk 1]
```diff
@@ -103,7 +103,7 @@ UPDATE msgs_msg SET
 		WHEN 
 			:status = 'E' 
 		THEN 
-			NOW() + (5 * (error_count+1) * interval '1 minutes') 
+			GREATEST(NOW() + (5 * (error_count+1) * interval '1 minutes'), :next_attempt::timestamp with time zone)
```
Comment (rowanseymour) [inline backends/rapidpro/status.go:106]: I'm not sure `:next_attempt::timestamp with time zone` is valid syntax - `:foo` syntax is just a way for the sqlx library to created queries that bind to struct fields but they ultimately get replaced with regular `$1`, `$2` etc param names.

**Comment by norkans7** [comment]
replaced by #411

File: backends/rapidpro/status.go (hunk 1 above)
Comment (norkans7) [reply backends/rapidpro/status.go:106]: I changed to use a string for interval that we can handle well easily in the SQL

Comment (norkans7) [reply backends/rapidpro/status.go:106]: #411

**Pull request closed without merging**